jujugui/static/gui/src/app/components/deployment-flow/deployment-flow.js

```diff
@@ -233,11 +233,11 @@ class DeploymentFlow extends React.Component {
   /**
     Store the provided SSH key in state.
 
-    @method _setSSHKey
-    @param {String} key The SSH key.
+    @method _setSSHKeys
```

Comment:
Oh, I'd misread your comment. Are we removing `@method` now? News to me. Should we do the whole file?